index.html

```diff
@@ -28,7 +28,7 @@
 <body>
     <div class="container header">
         <div class="header-left">
-            <h1>Indie Wiki Buddy</h1>
+            <h1><img src="img/favicon-64.png" width="40px" height="40px" align="center">     Indie Wiki Buddy</h1>
```

Comment:
The `align` attribute is deprecated for image elements, should use CSS instead. Can use `vertical-align: middle;` to vertically align.